src/services/findAllReferences.ts

```diff
@@ -510,6 +510,8 @@ namespace ts.FindAllReferences.Core {
         /** @param allSearchSymbols set of additinal symbols for use by `includes`. */
         createSearch(location: Node, symbol: Symbol, comingFrom: ImportExport | undefined, searchOptions: { text?: string, allSearchSymbols?: Symbol[] } = {}): Search {
             // Note: if this is an external module symbol, the name doesn't include quotes.
+            // Note: getLocalSymbolForExportDefault handles `export default class C {}`, but not `export default C` or `export { C as default }`.
+            // The other two forms seem to be handled downstream (i.e. special-casing the first form here appears to be intentional).

```

Comment:
This may be because we don't get a default export alias symbol here. See for example `skipPastExportOrImportSpecifier`.

Reply:
Supplemented the comment with that information.